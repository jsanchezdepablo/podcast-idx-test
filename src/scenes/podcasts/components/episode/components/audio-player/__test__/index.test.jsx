import React from "react";
import { shallow } from "enzyme";
import AudioPlayer from "../";

const MOCK_PROPS = {
  src: "src",
};

describe("AudioPlayer Component", () => {
  it("Matches snapshot", () => {
    const wrapper = shallow(<AudioPlayer {...MOCK_PROPS} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("Should stop playing", () => {
    const wrapper = shallow(<AudioPlayer {...MOCK_PROPS} />);
    wrapper.find(".audio__player").props().onPause();
  });

  it("Should start playing", () => {
    const wrapper = shallow(<AudioPlayer {...MOCK_PROPS} />);
    wrapper.find(".audio__player").props().onPlay();
  });
});
