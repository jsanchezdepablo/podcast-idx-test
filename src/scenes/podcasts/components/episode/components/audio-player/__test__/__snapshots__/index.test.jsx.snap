// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer Component Matches snapshot 1`] = `
<div
  className="audio__container"
>
  <audio
    className="audio__player"
    controls={true}
    onPause={[Function]}
    onPlay={[Function]}
  >
    <source
      src="src"
      type="audio/mpeg"
    />
    Your browser does not support the audio element.
  </audio>
</div>
`;
